drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -173,6 +177,27 @@ static int sbs_gauge_get_prop(const struct device *dev, struct fuel_gauge_get_pr
 	return rc;
 }
 
+#ifdef SBS_GAUGE_CUTOFF_SUPPORT
+static int sbs_gauge_do_battery_cutoff(const struct device *dev)
+{
+	int rc;
+	const struct sbs_gauge_config *cfg = dev->config;
+
+	if (cfg->cutoff_support == false) {
+		return -ENOTSUP;
+	}
+
+	for (int i = 0; i < ARRAY_SIZE(cfg->cutoff_payload); i++) {
+		rc = sbs_cmd_reg_write(dev, cfg->cutoff_reg_addr, cfg->cutoff_payload[0]);
```

Comment:
Looks like you need a test case where the payload value is not duplicated.

```suggestion
		rc = sbs_cmd_reg_write(dev, cfg->cutoff_reg_addr, cfg->cutoff_payload[i]);
```